backend/BackendMIIT/src/test/java/com/example/BackendMIIT/controller/DepartmentControllerTest.java

```diff
@@ -0,0 +1,21 @@
+package com.example.BackendMIIT.controller;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.test.web.servlet.MockMvc;
+
+@WebMvcTest(controllers = DepartmentController.class)
+@ExtendWith(MockitoExtension.class)
+public class DepartmentControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    public void getAllDepartmentsTest() throws Exception {
+        //TODO
```

Comment:
If you need to write the TODO comment, please, write down exactly what's going to be done. That clarification may also help other developers

Reply:
Ok i'll do that. But these todos was written for myself